improver_tests/utilities/test_cube_checker.py

```diff
@@ -217,37 +227,44 @@ def test_mismatch_in_first_cube(self):
     def test_mismatch_in_second_cube(self):
         """Test when finding a one-way mismatch, so that the first cube has
         a missing coordinate. This returns a list with the missing coordinate
-        name.l"""
-        cube = set_up_cube()
-        first_cube = cube.copy()
-        first_cube.remove_coord("time")
-        second_cube = cube.copy()
+        name."""
+        first_cube = self.cube.copy()
+        second_cube = self.cube.copy()
+        first_cube.remove_coord("realization")
         result = find_dimension_coordinate_mismatch(
             first_cube, second_cube, two_way_mismatch=False
         )
         self.assertIsInstance(result, list)
-        self.assertListEqual(result, ["time"])
+        self.assertListEqual(result, ["realization"])
 
     def test_two_way_mismatch(self):
         """Test when finding a two-way mismatch, when the first and second
         cube contain different coordinates."""
-        cube = set_up_cube()
-        first_cube = cube.copy()
-        first_cube.remove_coord("time")
-        second_cube = cube.copy()
+        first_cube = self.cube.copy()
+        second_cube = self.cube.copy()
         second_cube.remove_coord("realization")
+        second_cube = add_coordinate(second_cube, [10, 20], "height", "m")
+        iris.util.promote_aux_coord_to_dim_coord(second_cube, "height")
```

Comment:
This isn't needed - "height" is already a dimension.

Reply:
Removed.